sdks/js-sdk/README.md

```diff
@@ -0,0 +1,150 @@
+# Abstract
+
+This document outlines what is needed to provide a set of the SDK's that can be used for JavaScript platforms, specifically Cordova, React Native & Web.
+
+## Goals
+
+### Primary Goals
+
+* Provide SDKs (Core, Metrics, Auth etc..) for Mobile Developers to allow them to integrate Cordova, React Native & Web applications into Mobile Services.
```

Comment:
I want this to be true however.. should we move React Native to the secondary goals, along with other platforms like flutter?

Reply:
Moving RN to secondary. Flutter is dart-based but it definitely looks similar to RN, I'd love to see this sdk being 'upgraded' to a generic hybrid one instead of just JS, but best not to add this to the proposal yet.